fabric-dynamic-registry-api-v1/src/main/resources/fabric-dynamic-registry-api-v1.accesswidener

```diff
@@ -0,0 +1,3 @@
+accessWidener	v1	named
+accessible	class	net/minecraft/util/registry/DynamicRegistryManager$Info
+accessible  method  net/minecraft/util/registry/BuiltinRegistries   addRegistry (Lnet/minecraft/util/registry/RegistryKey;Lnet/minecraft/util/registry/MutableRegistry;Ljava/util/function/Supplier;Lcom/mojang/serialization/Lifecycle;)Lnet/minecraft/util/registry/MutableRegistry;
```

Comment:
be consistent with AW formatting lol

Reply:
Fixed in 9d7e2aeb898c4d6eefc509a3f79e8fa725700ef6